src/Symfony/Component/DependencyInjection/ContainerBuilder.php

```diff
@@ -94,9 +95,22 @@ class ContainerBuilder extends Container implements TaggedContainerInterface
      */
     private $usedTags = array();
 
+    private $context;
+
     private $compiled = false;
 
     /**
+     * @param ParameterBagInterface|null $parameterBag A ParameterBagInterface instance
+     * @param Context|null               $context      A Context instance, or null
```

Comment:
I mentioned it, because of https://www.phpdoc.org/docs/latest/guides/inheritance.html

AFAIK docblock inheritance happens by default from a _phpdoc_ pov.

> because a DocBlock for a class makes full use of the object-oriented principles that PHP offers and inherits the following information from the superclass

Ie. you'd inherit the wrong `@param` doc from `Container::__construct`. `@inheritdoc` is optional, however using it can be seen a best practice;

> it does make clear that an element has been explicitly documented (and thus not forgotten)

But using it inline is plain wrong :smile: (again from phpdoc pov).